### PR TITLE
Ipv6

### PR DESCRIPTION
**Pull request by Byon2**

### CODE REVIEW

**Review by kasgel — CHANGES_REQUESTED** [review]
_Prohibited networks check_ is missing IPv6 support

**Review by kasgel — CHANGES_REQUESTED** [review]
_Same source address_ is also missing IPv6 support

File: src/checks/consistency_glue_authoritative.py [hunk 1]
```diff
@@ -3,126 +3,54 @@
 #DNSHEALTH-14
 import dns, dns.resolver, dns.query, dns.name
 from dns.exception import DNSException
+#import src.checks.check_helpers as helpers
```
Comment (kasgel) [inline src/checks/consistency_glue_authoritative.py]: Unnecessary line?

File: src/checks/consistency_glue_authoritative.py [hunk 2]
```diff
@@ -136,52 +64,38 @@ def getGlueRecords(domain, list_of_name_servers):
...
+                        return {"result": False,"description":DESCRIPTION(), "details": "{0} could not be found in the glue records for ip addresses".format(ip["result"][0].to_text())}
 
                     results[i].remove(ip["result"][0].to_text())
 
         for _,value in results.items():
             if len(value) != 0:
-                return {"result": False, "description": "Check glue consistency", "details": "Extra addresses were found when queried the servers!({0})".format(value)}
-        return {"result": True,"description": "Check glue consistency", "details": "Sucess! There is consistency between glue and authoritative data!"}
+                return {"result": False, "description": DESCRIPTION(), "details": "Extra addresses were found when queried the servers!({0})".format(value)}
+        return {"result": True,"description":DESCRIPTION(), "details": "Sucess! There is consistency between glue and authoritative data!"}
```
Comment (kasgel) [inline src/checks/consistency_glue_authoritative.py]: `Sucess!` typo

File: src/main.py [hunk 3]
```diff
@@ -14,7 +14,8 @@
 # The user can enter a list of nameservers with the --ns param
 parser.add_argument('--ns', type=str, nargs='+', help="A list of NS servers to check.")
 parser.add_argument('--json', help="Output results from checks as JSON", action='count', default=0)
-
+# add argument to enable ipv6
+parser.add_argument('--ipv6', help ="Enables ipv6")
```
Comment (kasgel) [inline src/main.py]: Make it work the same way as `--json` for consistency. That entering only `--ipv6` enables it. Without `--ipv6` it defaults to ipv4. (no true or false value required for it)

**Review by kasgel — CHANGES_REQUESTED** [review]

**Comment by kasgel** [comment]
Also, there are no tests for ipv6. This needs to be corrected

**Comment by rarvid** [comment]
All tests should have the IPv6 flag (True, False). Preferably that there are tests for both IPv4 AND IPv6
@xyderos 
@hristog121 
@azzadm 
@KristapsKalnins

**Review by rarvid — APPROVED** [review]
LGTM

**Review by rarvid — APPROVED** [review]
changes approved

**Review by kasgel — APPROVED** [review]

**Pull request merged**